### PR TITLE
Feature/starter pack desktop

### LINKED ISSUE #157: Implement Desktop Version for Starter Pack page

### LINKED ISSUE #188: Starter pack: place all items 'from breeder' as inline

### LINKED ISSUE #189: Starter pack: apply designs an on the pet page

### PR DESCRIPTION
**Pull request by okhabr**
Closes #157 
Closes #188 
Closes #189
- implement desktop version of starter pack page
- improve styles of the starter pack page: 
   - place elements "from breeder" as inline
   - few paddings correction

### CODE REVIEW

**Review by ivan-bonk — APPROVED** [review]
👍

**Review by Pashkaua — APPROVED** [review]

File: src/pages/pet/components/observations/observations.module.scss [hunk 1]
```diff
@@ -17,5 +17,6 @@ $dark-grey: #8E8E93;
 }
 .plainText {
   font-size: 16px;
-  margin-bottom: 15px ;
+  margin-bottom: 15px;
+  margin-top: 6px;
 }
```
Comment (enheit) [inline src/pages/pet/components/observations/observations.module.scss]: nit: Don't forget to add empty line to the end of the file

File: src/pages/starter-pack/components/from-breeder/from-breeder.module.scss [hunk 2]
```diff
@@ -39,14 +40,9 @@
...
-    span {
-      margin: 3px 0;
-    }
-  }
 }
+@media (min-width: 768px) {
+  .fromBreeder{
+    padding-top: 20px;
+  }
+}
```
Comment (enheit) [inline src/pages/starter-pack/components/from-breeder/from-breeder.module.scss]: nit: Don't forget to add empty line to the end of the file

**Review by enheit — APPROVED** [review]

File: src/pages/starter-pack/components/from-breeder/from-breeder.module.scss (hunk 2 above)
Comment (okhabr) [reply src/pages/starter-pack/components/from-breeder/from-breeder.module.scss]: +

File: src/pages/pet/components/observations/observations.module.scss (hunk 1 above)
Comment (okhabr) [reply src/pages/pet/components/observations/observations.module.scss]: +

**Pull request merged**